docs/release-notes/v11.5.0.md

```diff
@@ -0,0 +1,217 @@
+---
+id: v11.5.0
+title: Version v11.5.0 Release Notes
+sidebar_label: v11.5.0
+image: "img/release-note-link-preview.png"
+---
+
+_Aug 31st, 2023_
+
+## Console
+
+### New Features
+
+#### Project Settings 
+
+Inside the 'Overview' area of a Project, you can now access a brand new section called 'Settings', where users with enough administrative permissions can view and manage the settings of their Project. This will speed-up and optimize the overall administration of Projects, which – until now – was only possible exclusively via CMS, making it no longer tedious or error-prone!
```

Comment:
```suggestion
Inside the 'Overview' Area of a Project, you can now access a brand new section called 'Settings', where users with enough administrative permissions can view and manage the settings of their Project. This will speed-up and optimize the overall administration of Projects, which – until now – was only possible exclusively via CMS, making it no longer tedious or error-prone!
```